### PR TITLE
Hw3 databases

### PR DESCRIPTION
**Pull request by sancho-sumy**

### CODE REVIEW

File: src/services/movies.services.ts [hunk 1]
```diff
@@ -0,0 +1,91 @@
+import { omit } from 'lodash';
+
+import { CustomError, Movie } from '../interfaces';
+import { GenreModel, MovieModel } from '../models';
+
+export async function getMovies() {
+    try {
+        const movies = await MovieModel.find();
+        return omit(movies, 'result');
```
Comment (vasylkivskyy) [inline src/services/movies.services.ts]: Why do we need omit from lodash in your functions?
It accepts objects, not array

File: src/validators/movies.validators.ts [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+                    return Promise.reject('Such movie is already exists!');
+                }
+            });
+        }),
+    body('description').trim().isLength({ min: 5 }).withMessage('Min length is 5 characters'),
+    body('releaseDate').trim().toDate(),
+    body('genre').isArray({ min: 1 }).withMessage('Please specify at least one genre'),
+];
+
+export const updateMovieValidator = [
```
Comment (vasylkivskyy) [inline src/validators/movies.validators.ts]: For improvements, you can combine this logic in two functions in one
Your second function has dublicated code from first

**Review by vasylkivskyy — CHANGES_REQUESTED** [review]

File: src/services/movies.services.ts (hunk 1 above)
Comment (sancho-sumy) [reply src/services/movies.services.ts]: Actually, we don't. 
I have used it before in one of the courses. Here I just copy/paste it and did not even think for what. 
Thanks to your comment I revised it.
Now it looks more correct.
Thank you very much!

File: src/validators/movies.validators.ts (hunk 2 above)
Comment (sancho-sumy) [reply src/validators/movies.validators.ts]: Validators has been refactored. I left only one movie/genre validator. MongoId is now checked by a separate validator.

File: src/services/movies.services.ts (hunk 1 above)
Comment (vasylkivskyy) [reply src/services/movies.services.ts]: image
You can also use the second paramater of mongoose methods to exclude some fields from response

Comment (sancho-sumy) [reply src/services/movies.services.ts]: Replaced `omit` by adding a second parameter to the MongoDB method.

**Review by vasylkivskyy — APPROVED** [review]

**Review by vasylkivskyy — APPROVED** [review]

**Pull request merged**